addons/account/models/account.py

```diff
@@ -714,16 +715,25 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
             if tax.price_include:
                 total_excluded -= tax_amount
                 base -= tax_amount
+                tax_base -= tax_amount
+                tax_excluded -= tax_amount
             else:
                 total_included += tax_amount
+                tax_included += tax_amount
 
             if tax.include_base_amount:
                 base += tax_amount
 
+            if tax.amount_type in ['division']:
```

Comment:
but maybe I am wrong on this one...I don't get the use case behind this option, but probably to help users for configuration.